packages/tools/src/tools/segmentation/RectangleScissorsTool.ts

```diff
@@ -150,20 +159,47 @@ class RectangleScissorsTool extends BaseTool {
       element,
       this.getToolName()
     );
-
     this.editData = {
       annotation,
-      segmentation,
       segmentIndex,
+      segmentationId,
       segmentsLocked,
       segmentColor,
-      segmentationId,
       viewportIdsToRender,
       handleIndex: 3,
       movingTextBox: false,
       newAnnotation: true,
       hasMoved: false,
-    };
+      segmentationRepresentationUID,
+    } as any;
+
+    if (isVolumeSegmentation(labelmapData as LabelmapSegmentationData)) {
```

Comment:
Seems like this code is getting repeated several times in each tool.  Could it be moved into a common top level class?

Reply:
I think we are fixing this in your strategy class